code/DepMapGeneEss/PlotGeneEss.R

```diff
@@ -0,0 +1,62 @@
+library(ggplot2)
+library(tidyverse)
+
+setwd("C:/Work/MatlabCode/components/human-GEM/Human-GEMDepMapEval/Human-GEM/code/DepMapGeneEss") #Modify to match your file system
+
+
+figPath = "figures/"
+
+
+############################################
+# Fig 1B - Gene essentiality
+############################################
+
+# load data
+# fns = c("data/geneEss_model1.txt",
+#         "data/geneEss_newalg.txt",
+#         "data/geneEss_newalg2.txt"
+#         )
+fns = c("data/geneEss_model1.txt")
+
+
+#names = c("tINIT","ftINIT 1+0", "ftINIT 1+1")
```

Comment:
```suggestion
```

Reply:
I'd like to keep that commented out code, will use it for the paper figure - this way we remember what to modify. My plan was to leave it like that for now and to tidy it up a bit when we know what the figure should look like.